docs/destinations/forwarding-events/snowbridge/concepts/scaling/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Scaling"
+sidebar_position: 150
+---
+
+# Scaling
+
+## Horizontal scaling
+
+Snowbridge is built to suit a horizontal scaling model, and you can safely deploy multiple instances of Snowbridge to consume the same input out-of-the-box. No addditional configuration or setup is required for the app to smoothly run across multiple instance/environment, compared to a single instance/environment.
+
+::note
+If you are using the Kinesis source, Snowbridge requires DynamoDB tables to manage consumers on the stream, as described in [the Kinesis source configuration section](/docs/destinations/forwarding-events/snowbridge/configuration/sources/kinesis.md).
```

Comment:
```suggestion
If you are using the Kinesis source, you will need to manually create a few DynamoDB tables as described in [the Kinesis source configuration section](/docs/destinations/forwarding-events/snowbridge/configuration/sources/kinesis.md). Snowbridge uses these tables to coordinate multiple instances consuming from the same stream.
```

Changing to highlight that a manual action is required. “Requires DynamoDB tables” does not convey that the user needs to intervene.